<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="navigationPanel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="inventoryButton_2">
        <property name="text">
         <string>Inventory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="employeeButton_2">
        <property name="text">
         <string>Employee</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="historyButton_2">
        <property name="text">
         <string>History</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logOutButton_2">
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="inventoryPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="inventoryNavigation">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="addItemButton_2">
            <property name="text">
             <string>Add Item</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="searchBar_2">
            <property name="placeholderText">
             <string>Product Name...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="searchButton_2">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QLabel" name="title_2">
            <property name="text">
             <string>INVENTORY</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="inventoryList">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTableWidget" name="products_2">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>156</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Product ID</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Quantity</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Price</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Category</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Actions</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="employeePage">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QFrame" name="employeeNavigation">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="addEmployeeButton">
            <property name="text">
             <string>Add Employee</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="searchBar_3">
            <property name="placeholderText">
             <string>Name...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="searchButton_3">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QLabel" name="title_3">
            <property name="text">
             <string>EMPLOYEE</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="employeeList">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QTableWidget" name="employees">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>186</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Employee ID</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Last Name</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>First Name</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Address</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Contact No.</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Date Hired</string>
             </property>
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="historyPage">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QFrame" name="historyNavigation">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="title_4">
            <property name="text">
             <string>TRANSACTION HISTORY</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="historyList">
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <property name="elideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <widget class="QWidget" name="PurchaseHistory">
          <attribute name="title">
           <string/>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QTableWidget" name="history">
             <column>
              <property name="text">
               <string>Receipt ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Customer Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>product</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Quantity</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Price</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Total Charge</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date Purchase</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="ServiceHistory">
          <attribute name="title">
           <string>Page</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTableWidget" name="history_2">
             <column>
              <property name="text">
               <string>Service ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Customer Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Device</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Service Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Technician</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Charge</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
